packages/sunburst/src/hooks.ts

```diff
@@ -68,6 +75,12 @@ export const useSunburst = <RawDatum>({
         // are going to be computed first
         const sortedNodes = sortBy(descendants, 'depth')
 
+        const innerRadiusOffset = radius * Math.min(innerRadiusRatio, 1)
+
+        const maxDepth = Math.max(...sortedNodes.map(n => n.depth))
+
+        const scale = d3ScaleRadial().domain([0, maxDepth]).range([innerRadiusOffset, radius])
```

Comment:
using d3 scaleLinear instead of scaleRadial would render better in my opinion

Reply:
scaleLinear looks quite different from the original implementation: 
![Screen Shot 2021-11-26 at 9 57 09 AM](https://user-images.githubusercontent.com/89929613/143617539-3b2b3ef5-5319-456c-8833-adfc6f625c1d.png)

Maybe we could update the sunburst props to include setters for the arc inner/outer radii. I'm not sure if this reaches a happy medium between customizability and ease of use, but my initial thought for the api would be something like: 
```
type GetInnerRadius = (currentNode: HierarchyRectangularNode<RawDatum>, nodes: HierarchyRectangularNode<RawDatum>[]) => number
```
@plouc Thoughts on this approach?